src/utils/rfq_blockchain_utils.ts

```diff
@@ -231,4 +248,58 @@ export class RfqBlockchainUtils {
     public async isWorkerReadyAsync(workerAddress: string, balance: BigNumber, gasPrice: BigNumber): Promise<boolean> {
         return isWorkerReadyAndAbleAsync(this._web3Wrapper, workerAddress, balance, gasPrice);
     }
+
+    /**
+     * Converts a @0x/web3-wrapper `TxData` object into an Ethers `TransactionRequest`.
+     *
+     * If the `from` address is not present in the `TxData`, the function uses the address
+     * of the Ethers Wallet passed to `rfqBlockchainUtils`
+     *
+     * If the `to` address is not present, the function uses the exchange proxy address.
+     */
+    public async transformTxDataToTransactionRequestAsync(
+        txOptions: Partial<TxData>,
+        chainId?: ChainId,
+        callData?: utils.BytesLike,
+    ): Promise<providers.TransactionRequest> {
+        let from = txOptions.from;
+        if (!from) {
+            if (!this._ethersWallet) {
+                throw new Error("An Ethers wallet must be provided if no 'from' address is specified");
+            }
+            from = await this._ethersWallet.getAddress();
+        }
+
+        return {
+            chainId,
+            data: callData,
+            from,
+            gasLimit: txOptions.gas instanceof BigNumber ? BigInt(txOptions.gas.toString()) : txOptions.gas,
```

Comment:
optional nit: can we add a comment calling out the renaming from `gas` to `gasLimit`?

Reply:
done